test/test_explore_multi_db.py

```diff
@@ -0,0 +1,15 @@
+import datetime
+
+from pagai.services.database_explorer import DatabaseExplorer, POSTGRES
+
+
+class TestExploration:
+    def test_explore(self, db_config):
+        explorer = DatabaseExplorer(db_config['handler'], db_config)
+        exploration = explorer.explore(table='patients', limit=2)
+
+        from pprint import pprint
```

Comment:
is there a reason why this import isn't at the global scope?

Reply:
haha no it's debug code I forgot to delete.